cmd/auto-updater/main_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main_test
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/versioncontrol"
+)
+
+func TestAutoUpdater(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	content, err := ioutil.ReadFile("testdata/fake-storagenode")
+	require.NoError(t, err)
+
+	tmpExec := ctx.File("storagenode")
+	err = ioutil.WriteFile(tmpExec, content, 0755)
+	require.NoError(t, err)
+
+	var mux http.ServeMux
+	content, err = ioutil.ReadFile("testdata/fake-storagenode.zip")
+	require.NoError(t, err)
+	mux.HandleFunc("/download", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write(content)
```

Comment:
maybe redundant comment but looks like jenkins linter is unhappy that this doesn't have an error check

Reply:
thanks, I missed that one